test/fixtures/fixture-commands.js

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+import meow from '../../index.js';
+
+const subcommand = (helpText, options = {}) => meow({
```

Comment:
```suggestion
const subcommand = (helpText, options) => meow({
```

The `options` should be guaranteed. Make sure the code guarantees it to exist.